ui/lib/core/addon/components/search-select.js

```diff
@@ -39,6 +39,7 @@ export default Component.extend({
   store: service(),
 
   onChange: () => {},
+  getSelectedValue: () => {},
```

Comment:
How is this method meaningfully different in usage than `onChange` ? I'm also sort of confused by the name of it, potentially changing to `onValueSelect` could be more clear. Also, we should add this to JSDocs